view/frontend/layout/checkout_index_index.xml

```diff
@@ -14,6 +14,30 @@
                                             <item name="children" xsi:type="array">
                                                 <item name="shippingAddress" xsi:type="array">
                                                     <item name="children" xsi:type="array">
+                                                        <item name="customer-email" xsi:type="array">
+                                                            <item name="children" xsi:type="array">
+                                                                <item name="before-login-form" xsi:type="array">
+                                                                    <item name="children" xsi:type="array">
+                                                                        <item name="bold-express-pays-before-email" xsi:type="array">
```

Comment:
Nitpick: "pays" here should be singular ("pay").

Reply:
Updated